core/modules/system/css/system.admin.css

```diff
@@ -221,6 +221,43 @@ span.admin-missing {
 .system-status-report .report-error .status-summary {
   color: #EC351C;
 }
+
+.system-status-report .status-description .item-list h3 {
+  font-size: 1em;
+}
+.system-status-report ul.requirements {
+  margin-left: 0; /* LTR */
+  padding-left: 0; /* LTR */
+}
+[dir="rtl"] .system-status-report ul.requirements {
+  margin-right: 0;
+  padding-right: 0;
+}
+.system-status-report ul.requirements li.status-ok,
+.system-status-report ul.requirements li.status-error {
+  list-style: none;
+  padding-left: 1em; /* LTR */
+  text-indent: -2ch;
+}
+[dir="rtl"] .system-status-report ul.requirements li.status-ok,
+[dir="rtl"] .system-status-report ul.requirements li.status-error {
+  padding-right: 1em;
+}
+.system-status-report ul.requirements li.status-ok:before {
+  content: "✓";
```

Comment:
A quick question: How about adding an actual space? `content:  "✓ ";`

Wouldn't that reduce the necessary CSS code? Or is that a problem with RTL languages?

Reply:
Sorry for taking that long to reply @indigoxela ...hmm 🤔 

I don't have a specific preference, but the CSS code reduction would basically only be the `padding-left`/`padding-right`. Right? ...is that worth it? 🤔 ...not sure about RTL - I'd have to test that and see. However, adding an actual space character in order to create CSS padding feels hackish to me, but searching google with "css content space instead of padding" didn't provide any specific articles saying whether this is bad practice or not. I'd say lets wait for more people with CSS/frontend experience to chime in. Happy to change once we have more feedback.